proxylib/redis/redcon.go

```diff
@@ -0,0 +1,213 @@
+package redis
```

Comment:
Needs copyright header

Reply:
Hi @aanm redcon.go is a slightly tweaked 3rd-party library as described above in the PR description.  Unfortunately, it seemed like it wouldn't work to just pull it in as a vendored library, though I could be wrong.    As such, I tried to make minimal changes to make it easier to backport any upstream fixes in the future.

Reply:
@danwent I don't think it matters as the original license is MIT.